website/search_migration/migrate.py

```diff
@@ -2,30 +2,46 @@
 # -*- coding: utf-8 -*-
 '''Migration script for Search-enabled Models.'''
 from __future__ import absolute_import
+
+import logging
 from modularodm.query.querydialect import DefaultQueryDialect as Q
+from scripts import utils as script_utils
 from website.models import Node
 from framework.auth import User
 import website.search.search as search
 
 from website.app import init_app
 
+logger = logging.getLogger(__name__)
+
 app = init_app("website.settings", set_backends=True, routes=True)
 
 
 def migrate_nodes():
-    for node in Node.find(Q('is_public', 'eq', True)
-            & Q('is_deleted', 'eq', False)):
+    n_iter = 0
+    nodes = Node.find(Q('is_public', 'eq', True) & Q('is_deleted', 'eq', False))
+    for node in nodes:
         node.update_search()
+        n_iter += 1
+
+    logger.info('Nodes migrated: {}'.format(n_iter))
 
 
 def migrate_users():
-    for user in User.find(Q('is_registered', 'eq', True)
-            & Q('date_confirmed', 'ne', None)):
-        user.update_search()
+    n_migr = 0
+    n_iter = 0
+    for user in User.find():
```

Comment:
This will require us to load every User, which isn't ideal. We want this migration to be as quick as possible, since Users won't be searchable until this finished. Can you add a `User.find_active` class method that will query for active users instead?